README.md

```diff
@@ -41,41 +41,43 @@ Multiple instances of the nDVP can run concurrently on the same host.  The allow
 
 2. After ensuring the correct version of Docker is installed, install and configure the NetApp Docker Volume Plugin.  Note, you will need to ensure that NFS and/or iSCSI is configured for your system.  See the installation instructions below for detailed information on how to do this.
 
-    ```bash
-    # download and unpack the application
-    wget https://github.com/NetApp/netappdvp/releases/download/v1.4.0/netappdvp-1.4.0.tar.gz
-    tar zxf netappdvp-1.4.0.tar.gz
-
-    # move to a location in the bin path
-    sudo mv netappdvp /usr/local/bin
-    sudo chown root:root /usr/local/bin/netappdvp
-    sudo chmod 755 /usr/local/bin/netappdvp
-
-    # create a location for the config files
-    sudo mkdir -p /etc/netappdvp
-
-    # create the configuration file, see below for more configuration examples
-    cat << EOF > /etc/netappdvp/ontap-nas.json
-    {
-        "version": 1,
-        "storageDriverName": "ontap-nas",
-        "managementLIF": "10.0.0.1",
-        "dataLIF": "10.0.0.2",
-        "svm": "svm_nfs",
-        "username": "vsadmin",
-        "password": "netapp123",
-        "aggregate": "aggr1"
-    }
-    EOF
-    ```
+	```bash
+# download and unpack the application
+wget -O /tmp/netappdvp-1.4.0.tar.gz https://github.com/NetApp/netappdvp/releases/download/v1.4.0/netappdvp-1.4.0.tar.gz 
+cd /tmp
+tar zxf netappdvp-1.4.0.tar.gz
+
+# move to a location in the bin path
+sudo mv netappdvp /usr/local/bin
+sudo chown root:root /usr/local/bin/netappdvp
+sudo chmod 755 /usr/local/bin/netappdvp
+
+# create a location for the config files
+sudo mkdir -p /etc/netappdvp
+
+# create the configuration file, see below for more configuration examples
+cat << EOF > /etc/netappdvp/ontap-nas.json
+{
+   "version": 1,
+   "storageDriverName": "ontap-nas",
+   "managementLIF": "10.0.0.1",
+   "dataLIF": "10.0.0.2",
+   "svm": "docker_svm",
+   "username": "vsadmin",
+   "password": "password",
+   "aggregate": "aggr",
+   "storagePrefix": "netappdvp_",
+}
+EOF
+	```    
```

Comment:
The indentation has been removed on these lines and some of the spaces have been converted to tabs.